pax-std/pax-std-primitives/src/text.rs

```diff
@@ -126,6 +126,15 @@ impl InstanceNode for TextInstance {
                     expanded_node.with_properties_unwrapped(|properties: &mut Text| {
                         let computed_tab = &expanded_node.layout_properties;
                         let (width, height) = computed_tab.bounds.get();
+                        let cp = expanded_node.get_common_properties();
+                        let cp = borrow!(cp);
+                        // send width/height only if common props exist, otherwise we are in "listening mode"
+                        // trying to infer width and height from the engine. To signal this we
+                        // send width/height = -1.0, telling chassis that "you tell me!".
+                        let (width, height) = (
+                            cp.width.get().is_some().then_some(width).unwrap_or(-1.0),
```

Comment:
I'm a bit confused here. It seems that width and height can be independently unspecified and thus be independently watched but we send back the ChassiResizeRequest with both dimensions always.

` let message = {
      "ChassiResizeRequest": {
          "id": id,
          "width": width,
          "height": height,
      }
  }`


Switching to a mode (aka enum) vs. a magic value (-1) should solve this. Either both height and width are set (FixedBoundingBox mode) or we're in AutoResize mode. This check essentially becomes an AND of both width and height to decide the mode.

Reply:
Main reason is that we can't listen independently for width/height changes, so might as well treat them as one, while the engine itself preferably (in my mind) allows autosizing in one direction or the other, or both, or none.